package.json

```diff
@@ -62,41 +62,42 @@
     "core-js": "3.22.8",
     "cross-spawn": "^7.0.3",
     "css-loader": "^6.7.3",
-    "css-minimizer-webpack-plugin": "^3.4.1",
+    "css-minimizer-webpack-plugin": "^5.0.0",
     "dotenv": "^16.0.3",
-    "dotenv-expand": "^8.0.3",
-    "eslint": "^8.35.0",
+    "dotenv-expand": "^10.0.0",
+    "eslint": "^8.39.0",
     "eslint-config-enact": "^4.1.4",
     "eslint-plugin-enact": "^1.0.2",
-    "eslint-plugin-jest": "^26.6.0",
+    "eslint-plugin-jest": "^27.2.1",
     "eslint-plugin-jsx-a11y": "^6.7.1",
     "eslint-plugin-react": "^7.32.2",
     "eslint-plugin-react-hooks": "^4.6.0",
-    "eslint-plugin-testing-library": "^5.10.2",
-    "eslint-webpack-plugin": "^3.2.0",
-    "expose-loader": "^3.1.0",
+    "eslint-plugin-testing-library": "^5.10.3",
+    "eslint-webpack-plugin": "^4.0.1",
+    "expose-loader": "^4.1.0",
     "file-loader": "^6.2.0",
-    "filesize": "^8.0.7",
-    "fs-extra": "^10.1.0",
-    "glob": "^7.2.3",
+    "filesize": "^10.0.7",
+    "fs-extra": "^11.1.1",
+    "glob": "^10.2.2",
     "global-modules": "^2.0.0",
-    "html-webpack-plugin": "^5.5.0",
+    "html-webpack-plugin": "^5.5.1",
     "identity-obj-proxy": "^3.0.0",
-    "jest": "^27.5.1",
+    "jest": "^29.5.0",
+    "jest-environment-jsdom": "^29.5.0",
```

Comment:
Why is this dependency needed?

Reply:
From jest v28+, if you are using JSDOM, jest-environment-jsdom package now must be installed separately:
guide: https://jestjs.io/docs/28.x/upgrading-to-jest28#jsdom
change: https://github.com/facebook/jest/pull/12354

Reply:
Please add this to the PR description.

Reply:
OK